R/rdeephaven/R/client_wrapper.R

```diff
@@ -0,0 +1,126 @@
+######################################## INCOMPLETE DOCS ########################################
+
+#' @title The Deephaven Client
+#' @description The Deephaven Client class is responsible for establishing and maintaining
+#' a connection to a running Deephaven server and facilitating basic server requests. This is
+#' the primary interface for interacting with the Deephaven server from R.
+#' 
+#' @section Connecting to the Server via Client
+#' 
+#' To connect to a Deephaven server, you must create a new DeephavenClient object via `Client$new(...)`,
+#' with the following arguments:
+#' 
+#' - `target`: The URL that the Deephaven server is running on.
+#' - `client_options`: A `ClientOptions` instance that contains the relevant information for connecting to your server.
+#'   Check the documentation on client options via `?ClientOptions`
+#' 
+#' @section Methods
+#' 
+#' - `$open_table(name)`: Looks for a table named 'name' on the server, and returns a Deephaven TableHandle reference
+#'    reference to that table if it exists. See the documentation on table handles via `?TableHandle` for more information.
+#' - `$import_table(table_object)`: Imports a new table to the Deephaven server and returns a Deephaven TableHandle
+#'    reference to the new table. Here, `table_object` can be an R Data Frame, an R Tibble, an Arrow Table, or an Arrow
+#'    RecordBatchReader. Note that this new table is not automatically bound to a variable name on the server.
+#'    See `?TableHandle` for more information.
+#' - `run_script(script)`: Runs a script on the server. The script must be in the language that the server console was
```

Comment:
```suggestion
#' - `$run_script(script)`: Runs a script on the server. The script must be in the language that the server console was
```
Other methods had the $